session106/XMap/XMap.xcodeproj/project.pbxproj

```diff
@@ -359,7 +375,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = PRG23MVE5M;
```

Comment:
We should not have a development team set, just set this back to None, once you're done.